qt/python/mantidqt/project/recovery/recoverygui/test/test_projectrecoverymodel.py

```diff
@@ -0,0 +1,138 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+
+import unittest
+import os
+import shutil
+import sys
+import time
+
+from mantidqt.project.recovery.recoverygui.projectrecoverymodel import ProjectRecoveryModel
+from mantidqt.project.recovery.projectrecovery import ProjectRecovery
+from mantid.simpleapi import CreateSampleWorkspace
+from mantid.api import AnalysisDataService as ADS
+from mantid.kernel import ConfigService
+
+if sys.version_info.major >= 3:
+    from unittest import mock
+else:
+    import mock
+
+
+class ProjectRecoveryModelTest(unittest.TestCase):
+    def setUp(self):
+        self.pr = ProjectRecovery(mock.MagicMock(), mock.MagicMock(), mock.MagicMock())
```

Comment:
Mocks should be moved to member variables, and have assertions in the tests

Reply:
None of these need assertions as they are passed in to this class and just need to be present at time of creation, I could have just as easily put None, but this is more robust for testing.

Reply:
If that is possible, I think it would be better to change them to `None`, as that would then fail a test that tries to use any of them, when it is not intended for them to be used. If for some reason they are used in the future, these tests should fail, and have to be updated to reflect the added functionality

Reply:
Will replace with None after discussion.